rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
+
+    book.rentals << self
+    person.rentals << self
+  end
+end
```

Comment:
- [ ] I couldn't test adding and displaying rentals due to the other errors. Kindly make sure it works as intended.